requirements/base.txt

```diff
@@ -35,22 +35,22 @@ mysqlclient==2.0.1        # via -r requirements/base.in
 newrelic==4.8.0.110       # via -r requirements/base.in, edx-django-utils
 openapi-codec==1.3.2      # via django-rest-swagger
 path.py==3.0.1            # via -r requirements/base.in
-pbr==5.4.5                # via stevedore
-psutil==1.2.1             # via edx-django-utils
+pbr==5.5.0                # via stevedore
+psutil==5.7.2             # via edx-django-utils
 pycparser==2.20           # via cffi
 pycryptodomex==3.9.8      # via pyjwkest
 pyjwkest==1.4.2           # via edx-drf-extensions
 pyjwt[crypto]==1.7.1      # via -r requirements/base.in, drf-jwt
-pymongo==3.10.1           # via edx-opaque-keys
+pymongo==3.11.0           # via edx-opaque-keys
```

Comment:
Many words, nothing that looks too scary though:
https://pymongo.readthedocs.io/en/stable/changelog.html#changes-in-version-3-11-0